### PR TITLE
docs: balsamic methods description in readthedocs

### PR DESCRIPTION
**Pull request by ashwini06**
### This PR:

Address: #892 

Added: Readthedocumentation part for balsamic methods 

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: docs/balsamic_filters.rst [hunk 1]
```diff
@@ -88,27 +90,28 @@ Following are the set of criterias applied for filtering vardict results. Applie
...
 
 **TNscope filtering (Tumor_normal)**
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To the variants in TNscope raw vcf file `SNV.somatic.$CASE_ID.tnscope.all.vcf.gz`, the following filters are applied and those that scored as `PASS` are considered to the final vcf file `SNV.somatic.$CASE_ID.tnscope.all.filtered.pass.vcf.gz`.
 
 *Total Depth (DP)*: Refers to the overall read depth from all target samples supporting the variant call
 
 ::
 
     DP(tumor) >= 10 || DP(normal) >= 10
```
Comment (keyvanelhami) [inline docs/balsamic_filters.rst:112]: I don't think the customers will understand the `||`sign. Lets wrote "or" instead

File: docs/balsamic_filters.rst [hunk 2]
```diff
@@ -176,6 +181,57 @@ The `TNscope <https://www.biorxiv.org/content/10.1101/250647v1.abstract>`_ algor
...
+**TNhaplotyper filtering (tumor_only)**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The variants in TNhaplotyper raw vcf file `SNV.somatic.$CASE_ID.tnhaplotyper.all.vcf.gz` are filtered for WGS interval regions; collected from gatk_bundles `<gs://gatk-legacy-bundles/b37/wgs_calling_regions.v1.interval_list>`_
+and the following filters are applied. The variants that scored as `PASS` are considered for `Merging of TNscope and TNhaplotyper results (tumor_only)`_
+
+*Total Depth (DP)*: Refers to the overall read depth from all target samples supporting the variant call
+
+::
+
+    DP(tumor) >= 10 || DP(normal) >= 10
```
Comment (keyvanelhami) [inline docs/balsamic_filters.rst:193]: I don't think the customers will understand the ||sign. Lets wrote "or" instead

File: docs/balsamic_filters.rst [hunk 3]
```diff
@@ -176,6 +181,57 @@ The `TNscope <https://www.biorxiv.org/content/10.1101/250647v1.abstract>`_ algor
...
+
+*Read Counts*: Count of reads in a given (F1R2, F2R1) pair orientation supporting the alternate allele and reference alleles
+
+::
+
+    ALT_F1R2 > 0, ALT_F2R1 > 0
+    REF_F1R2 > 0, REF_F2R1 > 0
+
+
+**Merging of TNscope and TNhaplotyper results (tumor_only)**
```
Comment (keyvanelhami) [inline docs/balsamic_filters.rst:228]: Is this the vcf-file uplaoded to Scout? If so, please add that text

File: docs/balsamic_filters.rst [hunk 4]
```diff
@@ -192,16 +248,16 @@ The following filter applies for both tumor-normal and tumor-only samples.
 
     minreads = 3,1,1
 
-Which means that at least `3` UMI tag groups should be ideally considered from both DNA strands, where a minimum of atleast `1` UMI tag group should exist in each of the single-stranded consensus reads.
+It means that at least `3` UMI tag groups should be ideally considered from both DNA strands, where a minimum of at least `1` UMI tag group should exist in each of the single-stranded consensus reads.
```
Comment (keyvanelhami) [inline docs/balsamic_filters.rst:251]: Maybe add a web-link with information regarding how UMI works? For those customers that don't have than info

File: docs/balsamic_filters.rst [hunk 5]
```diff
@@ -192,16 +248,16 @@ The following filter applies for both tumor-normal and tumor-only samples.
...
 
-*min_init_tumor_lod* : Log odds is the likelihood that the candidate mutation is real over the likelihood that the candidate mutation is a sequencing error before any read-based filters are applied.
-minimum log odds for the candidate selection. TNscope default: `4`
+*min_init_tumor_lod*: Log odds is the likelihood that the candidate mutation is real over the likelihood that the candidate mutation is a sequencing error before any read-based filters are applied.
+Minimum log-odds for the candidate selection. TNscope default: `4`
```
Comment (keyvanelhami) [inline docs/balsamic_filters.rst:254]: Is `TNscope default: 4` the value that we also use? If so, please clarify this

File: docs/balsamic_filters.rst [hunk 6]
```diff
@@ -192,16 +248,16 @@ The following filter applies for both tumor-normal and tumor-only samples.
...
 
 ::
 
     min_init_tumor_lod = 0.5
 
-*min_tumor_lod* : minimum log odds in the final call of variants. TNscope default: `6.3`
+*min_tumor_lod*: minimum log odds in the final call of variants. TNscope default: `6.3`
```
Comment (keyvanelhami) [inline docs/balsamic_filters.rst:260]: Is `TNscope default: 6.3` the value that we also use? If so, please clarify this

File: docs/balsamic_methods.rst [hunk 7]
```diff
@@ -0,0 +1,87 @@
+========================
+BALSAMIC METHODS
+========================
+
+Target Genome Analysis
+~~~~~~~~~~~~~~~~~~~~~~
+
+BALSAMIC :superscript:`1` (**version** = 8.2.8) was used to analyze the data from raw FASTQ files.
```
Comment (keyvanelhami) [inline docs/balsamic_methods.rst:8]: When looking at the text as markdown "Display the rich off", it looks very messy, but maybe that is only for me?

File: docs/balsamic_methods.rst [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+We first quality controlled FASTQ files using FastQC v0.11.9 :superscript:`2`.
+Adapter sequences and low-quality bases were trimmed using fastp v0.20.1 :superscript:`3`.
+Trimmed reads were mapped to the reference genome hg19 using BWA MEM v0.7.15 :superscript:`4`.
```
Comment (keyvanelhami) [inline docs/balsamic_methods.rst:11]: Will the version of the different tools be updated with `superscript: 4` ??

File: docs/balsamic_methods.rst [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+The resulted SAM files were converted to BAM files and sorted using samtools v1.12 :superscript:`5`.
+Duplicated reads were marked using Picard tools MarkDuplicate v2.25.0 :superscript:`6`
+and promptly quality controlled using CollectHsMetrics, CollectInsertSizeMetrics and CollectAlignmentSummaryMetrics functionalities.
+Results of the quality controlled steps were summarized by MultiQC v1.11 :superscript:`7`.
+Small somatic mutations were called for each sample using VarDict v2019.06.04 :superscript:`8`.
```
Comment (keyvanelhami) [inline docs/balsamic_methods.rst:16]: Please clarify "Small somatic mutations". VarDict also calls small INDELs and SVs?

File: docs/balsamic_methods.rst [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+Apart from the Vardict filters to report the variants, the called-variants were also further second filtered using criteria
+(*MQ >= 40, DP >= 100, VD >= 5, Minimum AF >= 0.007, Maximum AF < 1, GNOMADAF_popmax <= 0.005*).
+Only those variants that fulfilled the filtering criteria and scored as `PASS` in the vcf file were reported.
+Structural variants were called using Manta v1.6.0 :superscript:`9` and Delly v0.8.7 :superscript:`10`.
+Copy number aberrations were called using CNVkit v0.9.4 :superscript:`11`.
+The variant calls from CNVkit,Manta and Delly were merged using SVDB v2.6.0 :superscript:`12`.
```
Comment (keyvanelhami) [inline docs/balsamic_methods.rst:22]: "CNVkit, Manta", add a whitespace after comma

File: docs/balsamic_methods.rst [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+~~~~~~~~~~~~~~~~~~~~~
+BALSAMIC :superscript:`1` (**version** = 8.2.8) was used to analyze the data from raw FASTQ files.
+We first quality controlled FASTQ files using FastQC v0.11.9 :superscript:`2`.
+Adapter sequences and low-quality bases were trimmed using fastp v0.20.1 :superscript:`3`.
+Trimmed reads were mapped to the reference genome hg19 using sentieon-tools :superscript:`15`.
+The resulted SAM files were converted to BAM files and sorted using samtools v1.12 :superscript:`5`.
+Duplicated reads were marked using Picard tools MarkDuplicate v2.25.0 :superscript:`6`
+and promptly quality controlled using CollectMultipleMetrics and CollectWgsMetrics functionalities.
+Results of the quality controlled steps were summarized by MultiQC v1.11 :superscript:`7`.
+Small somatic mutations were called for each sample using Sentieon TNscope and TNhaplotyper :superscript:`16`.
```
Comment (keyvanelhami) [inline docs/balsamic_methods.rst:35]: Please clarify "Small somatic mutations". Sentieon also calls small INDELs and SVs?

File: docs/balsamic_methods.rst [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+The called-variants were also further second filtered using the criteria (DP(tumor,normal) >= 10; AD(tumor) >= 3; AF(tumor) >= 0.05, Maximum AF(tumor < 1;  GNOMADAF_popmax <= 0.001; normalized base quality scores >= 20, read_counts of alt,ref alle > 0).
+The filtered variants from TNscope and TNhaplotyper were merged using bcftools isec functionality to reduce the number of variants for tumor-only samples.
+Structural variants were called using Manta v1.6.0 :superscript:`9` and Delly v0.8.7 :superscript:`10`.
+Copy number aberrations were called using ascatNgs v4.5.0` :superscript:`17`.
```
Comment (keyvanelhami) [inline docs/balsamic_methods.rst:39]: `superscript:17` is not highlighted as "purple"

File: docs/balsamic_filters.rst [hunk 13]
```diff
@@ -176,6 +181,57 @@ The `TNscope <https://www.biorxiv.org/content/10.1101/250647v1.abstract>`_ algor
...
+
+    ALT_F1R2 > 0, ALT_F2R1 > 0
+    REF_F1R2 > 0, REF_F2R1 > 0
+
+
+**Merging of TNscope and TNhaplotyper results (tumor_only)**
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For tumor-only samples, to reduce the number of reported somatic variants,
+`TNscope filtering (tumor_only)`_ and `TNhaplotyper filtering (tumor_only)`_ results are merged using the `bcftools` intersect
+functionality. Only those variants called by both callers are confidently reported as the final filtered list of variants (`SNV.somatic.{CASE_ID}.tnscope.all.filtered.pass.vcf.gz`)
```
Comment (khurrammaqbool) [inline docs/balsamic_filters.rst]: The somatic variants from `TNscope filtered (tumor_only)`_ and `TNhaplotyper filtered (tumor_only)`_  are merged using the `bcftools intersect` command for tumor-only samples to reduce the number of reported somatic variants. Next, the somatic variants that are called by both variant-callers are reported as the final filtered list of variants (`SNV.somatic.{CASE_ID}.tnscope.all.filtered.pass.vcf.gz`). The final VCF constitutes a high confidence set of somatic variants.

File: docs/balsamic_filters.rst (hunk 4 above)
Comment (ashwini06) [reply docs/balsamic_filters.rst:252]: Indeed! There is a description for UMI workflow with [link](https://github.com/Clinical-Genomics/BALSAMIC/blob/docs/methods_description/docs/balsamic_filters.rst#sentieons-tnscope-1) in the beginning of **Sentieon's TNscope** (line 240)

File: docs/balsamic_filters.rst (hunk 5 above)
Comment (ashwini06) [reply docs/balsamic_filters.rst]: Sentieon's default is 4. But the value we use is in the next line `min_init_tumor_lod = 0.5`. To clarify, I added the same in words now

File: docs/balsamic_filters.rst (hunk 6 above)
Comment (ashwini06) [reply docs/balsamic_filters.rst]: Sentieon's default is 6.3 But the value we use is in the next line m`in_tumor_lod = 4.0` To clarify, I added the same in words now

File: docs/balsamic_methods.rst (hunk 8 above)
Comment (ashwini06) [reply docs/balsamic_methods.rst:11]: The purpose of superscript is to add citations numbers to the tools we used. The numbers corresponds to the order of  citation in the *References* section. The version numbers are not updated and manually need to be changed in this document, whenever any of the tools are updated in BALSAMIC.

File: docs/balsamic_methods.rst (hunk 11 above)
Comment (ashwini06) [reply docs/balsamic_methods.rst]: Fixed!

File: docs/balsamic_methods.rst (hunk 12 above)
Comment (ashwini06) [reply docs/balsamic_methods.rst]: Fixed!

File: docs/balsamic_methods.rst (hunk 10 above)
Comment (ashwini06) [reply docs/balsamic_methods.rst]: Fixed!

File: docs/balsamic_filters.rst (hunk 2 above)
Comment (ashwini06) [reply docs/balsamic_filters.rst]: Fixed!

File: docs/balsamic_filters.rst [hunk 14]
```diff
@@ -3,32 +3,35 @@ BALSAMIC Variant Calling Algorithms
 ***********************************
 
 In BALSAMIC, various bioinfo tools are integrated for reporting somatic and germline variants. Also, the choice of these tools differs between the type of analysis,
-for eg: `Target Genome Analysis (TGA)` or `Whole Genome Sequencing (WGS)`. Various filters (Pre-call filtering and Post-call filtering) are applied at different levels to report high-confidence variant calls.
+eg: `Target Genome Analysis (TGA)` or `Whole Genome Sequencing (WGS)`. Various filters (Pre-call and Post-call filtering) are applied at different levels to report high-confidence variant calls.
```
[the remark this answers is not in the document]
Comment (ashwini06) [reply docs/balsamic_filters.rst]: Fixed!

File: docs/balsamic_filters.rst [hunk 15]
```diff
@@ -88,27 +90,28 @@ Following are the set of criterias applied for filtering vardict results. Applie
...
 **Sentieon's TNscope**
 =======================
 
-BALSAMIC utilizes `TNscope` algorithm for the variant calling of somatic SNV/INDELS in WGS samples.
+BALSAMIC utilizes the `TNscope` algorithm for the variant calling of somatic SNV/INDELS in WGS samples.
 The `TNscope <https://www.biorxiv.org/content/10.1101/250647v1.abstract>`_ algorithm performs the somatic variant calling on the tumor-normal or the tumor-only samples, using a Haplotyper algorithm.
 
 **TNscope filtering (Tumor_normal)**
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To the variants in TNscope raw vcf file `SNV.somatic.$CASE_ID.tnscope.all.vcf.gz`, the following filters are applied and those that scored as `PASS` are considered to the final vcf file `SNV.somatic.$CASE_ID.tnscope.all.filtered.pass.vcf.gz`.
```
[the remark this answers is not in the document]
Comment (ashwini06) [reply docs/balsamic_filters.rst]: Fixed!

File: docs/balsamic_filters.rst [hunk 16]
```diff
@@ -129,14 +132,16 @@ The `TNscope <https://www.biorxiv.org/content/10.1101/250647v1.abstract>`_ algor
 
 **TNscope filtering (tumor_only)**
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The variants in TNscope raw vcf file `SNV.somatic.$CASE_ID.tnscope.all.vcf.gz` are filtered for WGS interval regions; collected from gatk_bundles `<gs://gatk-legacy-bundles/b37/wgs_calling_regions.v1.interval_list>`_
```
[the remark this answers is not in the document]
Comment (ashwini06) [reply docs/balsamic_filters.rst]: Modified the text to add clarity

File: docs/balsamic_filters.rst [hunk 17]
```diff
@@ -3,32 +3,35 @@ BALSAMIC Variant Calling Algorithms
...
 
 ..  figure:: images/vcf_filters.png
     :width: 500px
 
-    Pre-call filters applied by the `Vardict` variant-caller is listed out in the VCF header
+    Pre-call filters applied by the `Vardict` variant-caller is listed in the VCF header.
 
 
-In the VCF file, `FILTER` status is `PASS` if this position has passed all filters, i.e., a call is made at this position. Otherwise,
+In the VCF file, the `FILTER` status is `PASS` if this position has passed all filters, i.e., a call is made at this position. Otherwise,
```
[the remark this answers is not in the document]
Comment (ashwini06) [reply docs/balsamic_filters.rst:19]: @khurrammaqbool Not sure of this comment where you intend to add this text. I added it at the end of the paragraph. Please check it again

**Comment by ashwini06** [comment]
@keyvanelhami @khurrammaqbool: Thanks for the nice suggestions. Fixed all your comments. Please review it again and approve if everything is fine!

File: docs/balsamic_filters.rst (hunk 17 above)
[the remark this answers is not in the document]
Comment (khurrammaqbool) [reply docs/balsamic_filters.rst:19]: It is fine at the end.

File: docs/balsamic_filters.rst [hunk 18]
```diff
@@ -3,32 +3,36 @@ BALSAMIC Variant Calling Algorithms
...
-for eg: `Target Genome Analysis (TGA)` or `Whole Genome Sequencing (WGS)`. Various filters (Pre-call filtering and Post-call filtering) are applied at different levels to report high-confidence variant calls.
+eg: `Target Genome Analysis (TGA)` or analysis of `Whole Genome Sequencing (WGS)`. Various filters (Pre-call and Post-call filtering) are applied at different levels to report high-confidence variant calls.
 
-* **Pre-call filtering** is where the variant-calling tool decides not to call a variant line to the VCF file, if the default filters did not pass the criteria. The set of default filters differs between the various variant-calling algorithms.
+**Pre-call filtering** is where the variant-calling tool decides not to add a variant to the VCF file if the default filters of the variant-caller did not pass the filter criteria. The set of default filters differs between the various variant-calling algorithms.
 
 To know more about the pre-call filters used by the variant callers, please have a look at the VCF header of the particular variant-calling results.
```
Comment (khurrammaqbool) [inline docs/balsamic_filters.rst:10]: please have a look at the VCF header of the particular variant-calling results to know more about the pre-call filters used by the variant callers

File: docs/balsamic_methods.rst [hunk 19]
```diff
@@ -0,0 +1,87 @@
...
+========================
+
+Target Genome Analysis
+~~~~~~~~~~~~~~~~~~~~~~
+
+BALSAMIC :superscript:`1` (**version** = 8.2.8) was used to analyze the data from raw FASTQ files.
+We first quality controlled FASTQ files using FastQC v0.11.9 :superscript:`2`.
+Adapter sequences and low-quality bases were trimmed using fastp v0.20.1 :superscript:`3`.
+Trimmed reads were mapped to the reference genome hg19 using BWA MEM v0.7.15 :superscript:`4`.
+The resulted SAM files were converted to BAM files and sorted using samtools v1.12 :superscript:`5`.
```
Comment (khurrammaqbool) [inline docs/balsamic_methods.rst:12]: The resulted SAM files are converted to BAM files and sorted using samtools v1.12 :superscript:`5`.

File: docs/balsamic_methods.rst [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+Duplicated reads were marked using Picard tools MarkDuplicate v2.25.0 :superscript:`6`
```
Comment (khurrammaqbool) [inline docs/balsamic_methods.rst:13]: Duplicated reads are marked using Picard tools MarkDuplicate v2.25.0 :superscript:`6`

File: docs/balsamic_methods.rst [hunk 21]
```diff
@@ -0,0 +1,87 @@
...
+and promptly quality controlled using CollectHsMetrics, CollectInsertSizeMetrics and CollectAlignmentSummaryMetrics functionalities.
+Results of the quality controlled steps were summarized by MultiQC v1.11 :superscript:`7`.
```
Comment (khurrammaqbool) [inline docs/balsamic_methods.rst:15]: Results of the quality controlled steps are then summarized by MultiQC v1.11 :superscript:`7`.

File: docs/balsamic_methods.rst [hunk 22]
```diff
@@ -0,0 +1,87 @@
...
+Small somatic mutations (SNVs and INDELs) were called for each sample using VarDict v2019.06.04 :superscript:`8`.
```
Comment (khurrammaqbool) [inline docs/balsamic_methods.rst:16]: Small somatic mutations (SNVs and INDELs) are called for each sample using VarDict v2019.06.04 :superscript:`8`.

File: docs/balsamic_methods.rst [hunk 23]
```diff
@@ -0,0 +1,87 @@
...
+Apart from the Vardict filters to report the variants, the called-variants were also further second filtered using criteria
+(*MQ >= 40, DP >= 100, VD >= 5, Minimum AF >= 0.007, Maximum AF < 1, GNOMADAF_popmax <= 0.005*).
+Only those variants that fulfilled the filtering criteria and scored as `PASS` in the vcf file were reported.
```
Comment (khurrammaqbool) [inline docs/balsamic_methods.rst:19]: Only those variants that fulfill the filtering criteria and are scored as `PASS` in the VCF file are reported.

File: docs/balsamic_methods.rst [hunk 24]
```diff
@@ -0,0 +1,87 @@
...
+Structural variants were called using Manta v1.6.0 :superscript:`9` and Delly v0.8.7 :superscript:`10`.
```
Comment (khurrammaqbool) [inline docs/balsamic_methods.rst:20]: Structural variants are called using Manta v1.6.0 :superscript:`9` and Delly v0.8.7 :superscript:`10`.

File: docs/balsamic_methods.rst [hunk 25]
```diff
@@ -0,0 +1,87 @@
...
+Copy number aberrations were called using CNVkit v0.9.4 :superscript:`11`.
+The variant calls from CNVkit, Manta and Delly were merged using SVDB v2.6.0 :superscript:`12`.
+All variants were annotated using Ensembl VEP v100.2 :superscript:`13`. Vcfanno v0.3.3 :superscript:`14` annotated variants for their population allele frequency from gnomAD v2.1.1 :superscript:`18`.
```
Comment (khurrammaqbool) [inline docs/balsamic_methods.rst:23]: Copy number aberrations are called using CNVkit v0.9.4 :superscript:`11` for TGA tumor normal samples.
The variant calls from CNVkit, Manta and Delly are then merged using SVDB v2.6.0 :superscript:`12`.
All variants are annotated using Ensembl VEP v100.2 :superscript:`13`. Only SNVs are annotated using Vcfanno v0.3.3 :superscript:`14` annotated variants for their population allele frequency from gnomAD v2.1.1 :superscript:`18`.

File: docs/balsamic_filters.rst [hunk 26]
```diff
@@ -3,32 +3,36 @@ BALSAMIC Variant Calling Algorithms
...
     :width: 500px
 
-    Pre-call filters applied by the `Vardict` variant-caller is listed out in the VCF header
+    Pre-call filters applied by the `Vardict` variant-caller is listed in the VCF header.
 
 
-In the VCF file, `FILTER` status is `PASS` if this position has passed all filters, i.e., a call is made at this position. Otherwise,
+In the VCF file, the `FILTER` status is `PASS` if this position has passed all filters, i.e., a call is made at this position. Otherwise,
 if the site has not passed all filters, a semicolon-separated list of codes for filters that fail. e.g., `p8;pSTD` might
 indicate that at this site, the mean position in reads is less than 8 and position in reads has a standard deviation of 0.
```
Comment (khurrammaqbool) [inline docs/balsamic_filters.rst:21]: Please rephrase or elaborate. I did not understand it clearly, so I could not rephrase it.

**Comment by ashwini06** [comment]
> Some general comments below: There are parts of the text that may need rephrasing or elaboration, I marked the few, but take a look at the grammar and fragmentation of the sentences. I suggest that the workflow should be described in present tense not past tense because it is describing the workflow not the analysis. I gave a few examples in the comments with the text. A minor comment, please use VCF instead of vcf where applicable.

@khurrammaqbool Thanks! For balsamic methods, it is intended as a copy-text for the customers into their materials and method section for their publications. Usually, we write the done analysis in past tense or present. For your ref: link to previous document which we sent to the customer https://docs.google.com/document/d/1zbC9mGpfABVdLK-Q0Bg7ILHQCKaeDgHKDqm6D4HBUpo/edit.

I try to address the rest of comments

File: docs/balsamic_methods.rst (hunk 7 above)
Comment (keyvanelhami) [reply docs/balsamic_methods.rst:8]: It looks like this for me. Should it be like this? If so, please resolve the  comment :)
Screen Shot 2022-04-14 at 9 24 20 AM

File: docs/balsamic_methods.rst [hunk 27]
```diff
@@ -0,0 +1,87 @@
+========================
+BALSAMIC METHODS
+========================
+
+Target Genome Analysis
+~~~~~~~~~~~~~~~~~~~~~~
+
+BALSAMIC :superscript:`1` (**version** = 8.2.8) was used to analyze the data from raw FASTQ files.
+We first quality controlled FASTQ files using FastQC v0.11.9 :superscript:`2`.
```
[the remark this answers is not in the document]
Comment (ashwini06) [reply docs/balsamic_methods.rst:9]: I would like to keep the methods in the past tense.

File: docs/balsamic_methods.rst (hunk 19 above)
Comment (ashwini06) [reply docs/balsamic_methods.rst:12]: I would like to keep the methods in the past tense.

File: docs/balsamic_methods.rst [hunk 28]
```diff
@@ -0,0 +1,87 @@
...
+Adapter sequences and low-quality bases were trimmed using fastp v0.20.1 :superscript:`3`.
+Trimmed reads were mapped to the reference genome hg19 using BWA MEM v0.7.15 :superscript:`4`.
+The resulted SAM files were converted to BAM files and sorted using samtools v1.12 :superscript:`5`.
+Duplicated reads were marked using Picard tools MarkDuplicate v2.25.0 :superscript:`6`
+and promptly quality controlled using CollectHsMetrics, CollectInsertSizeMetrics and CollectAlignmentSummaryMetrics functionalities.
+Results of the quality controlled steps were summarized by MultiQC v1.11 :superscript:`7`.
+Small somatic mutations (SNVs and INDELs) were called for each sample using VarDict v2019.06.04 :superscript:`8`.
+Apart from the Vardict filters to report the variants, the called-variants were also further second filtered using criteria
```
[the remark this answers is not in the document]
Comment (ashwini06) [reply docs/balsamic_methods.rst:17]: I would like to keep the methods in the past tense.

File: docs/balsamic_methods.rst (hunk 25 above)
Comment (ashwini06) [reply docs/balsamic_methods.rst:23]: I would like to keep the methods in the past tense.

File: docs/balsamic_filters.rst (hunk 26 above)
Comment (ashwini06) [reply docs/balsamic_filters.rst]: Edited the text now

**Pull request merged**